### DISCUSSION TITLE
Is it possible to retrieve the statuses_count for, say, the past month?

### DISCUSSION THREAD

**Discussion by ShredderWalnut** [mastodon/mastodon#17560] [Q&A] [answered]
Hello,

You can already retrieve the all-time `statuses_count` of an account. I would like to retrieve the statuses count of a set time period instead of the total count, e.g. the statuses count for the past week/month/quarter.

Is there a method/workaround for this? Thank you in advance!

**Comment by ClearlyClaire** [comment, marked as answer]
No, you'd have to recompute it from a SQL query, which is bound to be more expensive.
Something like the following should work:
```sql
SELECT count(*) FROM statuses WHERE account_id = 42 AND created_at BETWEEN '2021-01-01' AND '2021-02-01';
```

**Reply by ShredderWalnut** [reply]
> No, you'd have to recompute it

I expected so, since such a feature arguably isn't that useful for the API. 

Thank you for the confirmation and the SQL tip! <3
